.pre-commit-config.yaml

```diff
@@ -6,7 +6,8 @@ repos:
       - id: terraform-fmt
       - id: tflint
         args:
-          - "--config=__GIT_ROOT__/.tflint.hcl"
+          - "--module"
+          #- "--config=__GIT_ROOT__/.tflint.hcl"
```

Comment:
we don't need commented stuff out?

Reply:
We do not - since we're using `tflint` default configuration (I deleted `.tflint.hcl` since it should be empty)